virttest/utils_windows/drive.py

```diff
@@ -88,6 +88,19 @@ def shrink_volume(session, vol_id, size):
     session.cmd(shrink_cmd.format(script_path))
 
 
+def check_shrink_completion_event(session, vol_id):
```

Comment:
Hi @peixiu 
Sorry for replying so late.
From my understanding of https://learn.microsoft.com/en-us/windows-server/administration/windows-commands/wevtutil  and the purpose of this module `Windows drive utilities`.
```
{qe | query-events} <Path> [/lf:<Logfile>] [/sq:<Structquery>] [/q:<Query>] [/bm:<Bookmark>] [/sbm:<Savebm>] [/rd:<Direction>] [/f:<Format>] [/l:<Locale>] [/c:<Count>] [/e:<Element>]  
Reads events from an event log, from a log file, or using a structured query. By default, you provide a log name for <Path>. However, if you use the /lf option, then <Path> must be a path to a log file. If you use the /sq parameter, <Path> must be a path to a file that contains a structured query.
```
I think the function `check_shrink_completion_event` is close to the case level, not a framework API.
My idea is that we could create a function to warp the drive of querying event logs here, like 
`def query_event_logs(session)` or `def get_event_logs(session)`, and then we could use the returning info from the previous function to complete our checking in the test case.
I have another suggestion that we could put this code into the provider of tp-qemu since it looks like a tp-qemu's function and it doesn't relate to the Windows utils drive tool so it doesn't fit here.
Please let me know your opinion. Thanks.